src/github/client/github-queries.ts

```diff
@@ -78,7 +78,50 @@ export const getPullRequests = `query ($owner: String!, $repo: String!, $per_pag
     }
   }`;
 
-export const getCommits = (includeChangedFiles?: boolean) => `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String, $default_ref: String!) {
+export type getCommitsResponse = {
+  repository: {
+    defaultBranchRef: {
+      target: {
+        history: {
+          edges
```

Comment:
What's `edges`? It should have it's own type.

Reply:
array of commit objects, will add a type against.